rest/api_collections_test.go

```diff
@@ -132,6 +132,13 @@ func TestCollectionsPublicChannel(t *testing.T) {
 	})
 	defer rt.Close()
 
+	// grant 'b' so we can check a non-public but accessible doc as well
+	if rt.GetDatabase().OnlyDefaultCollection() {
+		require.NoError(t, rt.SetAdminChannels(username, rt.GetDatabase().Name, "b"))
+	} else {
+		require.NoError(t, rt.SetAdminChannels(username, rt.GetSingleKeyspace(), "b"))
+	}
```

Comment:
These actually are the same code, since `rt.GetSingleKeyspace()` will resolve to `db` if `SG_TEST_USE_DEFAULT_COLLECTION=true`. The other way to write this code which I'd be inclined to do for new tests is to remove the code to add a user in dbconfig but use `rt.CreateUser("alice", []string{"b"})`.

This is just informational for writing future tests, I'm happy to leave it.


```suggestion
	// grant 'b' so we can check a non-public but accessible doc as well
	require.NoError(t, rt.SetAdminChannels(username, rt.GetSingleKeyspace(), "b"))
```